samples/05_content_publishers/publishing_packages_as_web_layers.ipynb

```diff
@@ -1 +1 @@
\n", "                        <br/>Last Modified: June 26, 2017\n", "                        <br/>0 comments, 0 views\n", "                    </div>\n", "                </div>\n", "                "], "text/plain": ["<Item title:\"World_earthquakes_2000_2010\" type:Scene Package owner:arcgis_python>"]}, "execution_count": null, "metadata": {}, "output_type": "execute_result"}], "source": ["slpk_item = gis.content.add({}, data='data/World_earthquakes_2000_2010.slpk', folder='packages')\n", "slpk_item"]}, {"cell_type": "code", "execution_count": null, "metadata": {}, "outputs": [{"data": {"text/html": ["<div class=\"item_container\" style=\"height: auto; overflow: hidden; border: 1px solid #cfcfcf; border-radius: 2px; background: #f6fafa; line-height: 1.21429em; padding: 10px;\">\n", "                    <div class=\"item_left\" style=\"width: 210px; float: left;\">\n", "                       <a href='https://pythonapi.playground.esri.com/portal/home/item.html?id=a38d979ef3c0470194ab2e15eedd4c63' target='_blank'>\n", "                        <img src='https://pythonapi.playground.esri.com/portal/portalimages/desktopapp.png' class=\"itemThumbnail\">\n", "                       </a>\n", "                    </div>\n", "\n", "                    <div class=\"item_right\"     style=\"float: none; width: auto; overflow: hidden;\">\n", "                        <a href='https://pythonapi.playground.esri.com/portal/home/item.html?id=a38d979ef3c0470194ab2e15eedd4c63' target='_blank'><b>World_earthquakes_2000_2010</b>\n", "                        </a>\n", "                        <br/><img src='https://pythonapi.playground.esri.com/portal/home/js/jsapi/esri/css/images/item_type_icons/layers16.png' style=\"vertical-align:middle;\">Scene Layer by arcgis_python\n", "                        <br/>Last Modified: June 26, 2017\n", "                        <br/>0 comments, 0 views\n", "                    </div>\n", "                </div>\n", "                "], "text/plain": ["<Item title:\"World_earthquakes_2000_2010\" type:Scene Service owner:arcgis_python>"]}, "execution_count": null, "metadata": {}, "output_type": "execute_result"}], "source": ["slpk_layer = slpk_item.publish()\n", "slpk_layer"]}], "metadata": {"anaconda-cloud": {}, "esriNotebookRuntime": {"notebookRuntimeName": "ArcGIS Notebook Python 3 Standard", "notebookRuntimeVersion": "4.0"}, "kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.2"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": true, "toc_position": {}, "toc_section_display": true, "toc_window_display": true}}, "nbformat": 4, "nbformat_minor": 1}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    slpk_item = folder.add({}, data='data/World_earthquakes_2000_2010.slpk')
</pre><ol><li>need&nbsp;<code>title</code>&nbsp;and&nbsp;<code>item_type</code>&nbsp;as required key pairs in item_properties</li><li><code>data</code>&nbsp;--&gt;&nbsp;<code>file</code></li><li>need&nbsp;<code>.result()</code>&nbsp;for&nbsp;<code>Folder.add()</code>&nbsp;job result</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>